.github/workflows/pr-review-client-deploy.yml

```
@@ -0,0 +1,138 @@
+name: Deploy PR Review Client environment
+
+on:
+  pull_request:
+    branches:
+      - develop  
+    types:
+      - labeled
+      - opened
+      - reopened
+      - synchronize
+
+env:
+  AWS_DEFAULT_REGION: ca-central-1
+  FUNCTION_NAME: "forms-client-pr"
+  IMAGE: forms/client
+  REGISTRY: 687401027353.dkr.ecr.ca-central-1.amazonaws.com/pr_review
+  ROLE_ARN: arn:aws:iam::687401027353:role/forms-lambda-client
+
+jobs:
+  build-and-push-container:
+    if: ${{ !contains(github.event.pull_request.labels.*.name, 'Renovate') }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set envs
+        run: echo "PR_NUMBER=$(jq --raw-output .pull_request.number "$GITHUB_EVENT_PATH")" >> $GITHUB_ENV
+
+      - name: Checkout
+        uses: actions/checkout@dc323e67f16fb5f7663d20ff7941f27f5809e9b6 # v2.6.0
+
+      - name: Configure aws credentials using OIDC
+        uses: aws-actions/configure-aws-credentials@e1e17a757e536f70e52b5a12b2e8d1d1c60e04ef # v2.0.0
+        with:
+          role-to-assume: arn:aws:iam::687401027353:role/platform-forms-client-apply
+          role-session-name: TFApply
+          aws-region: ${{ env.AWS_DEFAULT_REGION }}
+
+      - name: Login to ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@261a7de32bda11ba01f4d75c4ed6caf3739e54be # v1.5.3
+
+      - name: Move dockerignore
+        run: |
+          mv ci/Dockerfile.lambda.dockerignore .
+
+      - name: Build Docker image
+        run: |
+          docker build -t ${{ env.REGISTRY }}/${{ env.IMAGE }}:${{ env.PR_NUMBER }} \
+            --build-arg GITHUB_SHA_ARG=$GITHUB_SHA \
+            --build-arg COGNITO_APP_CLIENT_ID=$COGNITO_APP_CLIENT_ID \
+            --build-arg COGNITO_USER_POOL_ID=$COGNITO_USER_POOL_ID .
```

Can you add the -f arg with the custom Dockerfile we're using:
```suggestion
          docker build -t ${{ env.REGISTRY }}/${{ env.IMAGE }}:${{ env.PR_NUMBER }} \
            -f Dockerfile.pr \
            --build-arg GITHUB_SHA_ARG=$GITHUB_SHA \
            --build-arg COGNITO_APP_CLIENT_ID=$COGNITO_APP_CLIENT_ID \
            --build-arg COGNITO_USER_POOL_ID=$COGNITO_USER_POOL_ID .
```